io.github.qalculate.Qalculate.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.qalculate.Qalculate
+runtime: org.gnome.Platform
+runtime-version: '3.32'
```

Comment:
Update runtime to 3.34

Reply:
When this application was submitted, the GNOME runtime was up to date. I've updated it to `3.32`, then to `3.34`. I see no point for further updates if this won't be merged soon.